### PR TITLE
Create first servlet with example content

### PR DESCRIPTION
**Pull request by jessdejong**
Followed the step-1 tutorial to modify my first servlet.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -20,13 +20,15 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+
```
Comment (estep-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:23]: nit: remove line (java style)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -20,13 +20,15 @@
...
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    final String myName = "Jess DeJong";
```
Comment (estep-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Having a final in a method (while allowed), is very uncommon. It doesn't change based on any method input, so this should be a class-level static:

`private static final String MY_NAME = "Jess DeJong";`

This goes outside the method right below the class declaration line. Also note the variable name style.

Comment (jessdejong) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Thanks, that makes sense! I'll make the change.

**Comment by jessdejong** [comment]
Commits (addressing comments):
Fix placement and style of final string variable,
and remove unnecessary blank line

**Pull request merged**